LISA/Executor.cpp

```diff
@@ -311,7 +313,11 @@ void Executor::doInstall(std::string type,
 {
     INFO("url=", url, " appName=", appName, " cat=", category);
 
-    // TODO check authentication method
+    auto authMethod = getAuthenticationMethod("", "", "");
+    if(NONE != authMethod) {
```

Comment:
Shouldn't header file be included for enum visibility?

Reply:
It was lost after merging to latest changes.